[2024-12-31T10:51:35.123+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.call_clear_all_data manual__2024-12-31T10:51:32.198725+00:00 [queued]>
[2024-12-31T10:51:35.137+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.call_clear_all_data manual__2024-12-31T10:51:32.198725+00:00 [queued]>
[2024-12-31T10:51:35.138+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2024-12-31T10:51:35.663+0000] {taskinstance.py:2191} INFO - Executing <Task(SQLExecuteQueryOperator): call_clear_all_data> on 2024-12-31 10:51:32.198725+00:00
[2024-12-31T10:51:35.672+0000] {standard_task_runner.py:60} INFO - Started process 1544 to run task
[2024-12-31T10:51:35.674+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'call_clear_all_data', 'manual__2024-12-31T10:51:32.198725+00:00', '--job-id', '96', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmpyotcy4bx']
[2024-12-31T10:51:35.676+0000] {standard_task_runner.py:88} INFO - Job 96: Subtask call_clear_all_data
[2024-12-31T10:51:35.732+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_data.call_clear_all_data manual__2024-12-31T10:51:32.198725+00:00 [running]> on host 403e7aadd7ee
[2024-12-31T10:51:35.806+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='agushchin' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='call_clear_all_data' AIRFLOW_CTX_EXECUTION_DATE='2024-12-31T10:51:32.198725+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-31T10:51:32.198725+00:00'
[2024-12-31T10:51:35.808+0000] {sql.py:276} INFO - Executing: CALL stage.clear_all_data()
[2024-12-31T10:51:35.818+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2024-12-31T10:51:35.829+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2024-12-31T10:51:35.846+0000] {sql.py:450} INFO - Running statement: CALL stage.clear_all_data(), parameters: None
[2024-12-31T10:51:35.875+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=call_clear_all_data, execution_date=20241231T105132, start_date=20241231T105135, end_date=20241231T105135
[2024-12-31T10:51:35.928+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-12-31T10:51:35.951+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
